test/utils/helpers.go

```diff
@@ -244,7 +247,9 @@ func ConfigureEntitySelectorAndWait(
 }
 
 // CheckCloudResourceNetworkPolicies checks NetworkPolicies has been applied to cloud resources.
-func CheckCloudResourceNetworkPolicies(k8sClient client.Client, kind, namespace string, ids []string, anps []string) error {
+func CheckCloudResourceNetworkPolicies(
```

Comment:
Following the same format as the function right above it. Do you want to change both? For function comments, I don't think this need a change, since its doing the same thing, just have a difference method for a special case.